packages/checkout-common/package.json

```
@@ -6,6 +6,9 @@
   "module": "dist/esm/index.js",
   "types": "./dist/index.d.ts",
   "scripts": {
+    "build": "rollup -c",
```

```suggestion
    "build": "NODE_ENV=production rollup -c",
```

We rely on this inside rollup.config.js